src/org/zaproxy/zap/authentication/PostBasedAuthenticationMethodType.java

```diff
@@ -278,13 +278,19 @@ public WebSession authenticate(SessionManagementMethod sessionManagementMethod,
 			// Prepare login message
 			HttpMessage msg;
 			try {
-				msg = prepareRequestMessage(cred);
 				// Make sure the message will be sent with a good WebSession that can record the changes
 				if (user.getAuthenticatedSession() == null)
 					user.setAuthenticatedSession(sessionManagementMethod.createEmptyWebSession());
+				
+				HttpMessage loginMsgToRenewCookie = new HttpMessage(new URI(loginRequestURL, true));
```

Comment:
interesting, so is it desired to set the default value as login URL?

Reply:
Does it make sense to set login URL field value as the default/initial value of login page URL field?

Reply:
Yeah, when setting up it can default to the same URL. (Sorry, somehow I missed your previous comment.)

Reply:
Thanks
(No problem, I hope it’s the one that I have deleted)

Reply:
Yeah, was referring to that one :)

Thinking better maybe we should leave this comment for a follow up change? The other comment does not require much changes so this could be merged sooner and fix #1370 which was the purpose all along.

Reply:
Nice, then I will address the second comment in this PR and will open a separate PR for this comment.

Reply:
Thank you.